pkg/skaffold/initializer/init.go

```diff
@@ -235,10 +245,47 @@ func processCliArtifacts(artifacts []string) ([]builderImagePair, error) {
 			return nil, fmt.Errorf("malformed artifact provided: %s", artifact)
 		}
 
-		pairs = append(pairs, builderImagePair{
-			Builder:   docker.Docker(parts[0]),
-			ImageName: parts[1],
-		})
+		// Parses JSON in the form of: {"name":"Name of Builder","payload": {...}}
+		// Parse name field first to determine builder type
+		nameCheck := struct {
+			Name    string      `json:"name"`
```

Comment:
You should be able to remove the json annotations here since you are just using the default names of every field

Reply:
Ah ok cool, assumed it was case sensitive.

Reply:
You're right. My comment is rubbish. Ignore it

Reply:
Is it? The tests still passed. Hmm.

Reply:
Oh I see, json.Unmarshal prefers an exact match, but checks for case insensitive matches if none are found. I'll change it back.